src/main/scala/DataBaseConn.scala

```diff
@@ -0,0 +1,57 @@
+
+//DataBaseConnection
+
+object DataBaseConn{
+  import cats.effect.IO
+  import doobie._
+  import doobie.implicits._
+
+  import scala.concurrent.ExecutionContext
+
+  implicit val cs = IO.contextShift(ExecutionContext.global)
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql:browserhistory", "postgres", ""
+  )
+
+  def findUserByName(username: String)  = {
```

Comment:
Can we do an interface containing those methods and the actual implementation to be in an implementation of that interface?